app/lib/api_constraints.rb

```diff
@@ -5,6 +5,6 @@ def initialize(options)
   end
 
   def matches?(req)
-    @default || req.headers["Accept"].include?("application/vnd.example.v#{@version}")
+    @default || req.headers["Accept"]&.include?("application/vnd.forem.api-v#{@version}+json")
```

Comment:
Two identical routes exist for each API endpoint (i.e. `GET /api/articles`) and the `ApiConstraint` class here matches V1 based on the `ACCEPT` header, with V0 as the default (accepts both `application/json` and `application/vnd.forem.api-v0+json`).

This means API consumers will see absolutely no difference (everything will continue to work with `V0`) until they pass in `ACCEPT: application/vnd.forem.api-v1+json`.

Tests at `spec/requests/api/v0/...` remain unchanged and continue to pass, while tests at `spec/requests/api/v1/...` require the new accept header to work.